src/uct/ib/mlx5/rc/rc_mlx5_iface.c

```diff
@@ -527,7 +553,10 @@ static ucs_status_t uct_rc_mlx5_iface_preinit(uct_rc_mlx5_iface_common_t *iface,
 #else
     iface->tm.enabled                = 0;
 #endif
-    init_attr->cq_len[UCT_IB_DIR_RX] = rc_config->super.rx.queue_len;
+    init_attr->cq_len[UCT_IB_DIR_RX] =
+            uct_rc_mlx5_iface_is_srq_msg_based(iface) ?
+                    UCT_RC_MLX5_IFACE_MSG_BASED_SRQ_CQ_LEN :
+                    rc_config->super.rx.cq_len;
```

Comment:
shouldn't it default to rx.queue_len? what if it has other value? 
in msg_based case it should be multiplied by max number of messages per WQE i.e. number of strides